xmodule/tests/test_library_sourced_block.py

```diff
@@ -2,63 +2,53 @@
 Tests for Source from Library XBlock
 """
 
-from openedx.core.djangoapps.content_libraries.tests.base import ContentLibrariesRestApiTest
-from common.djangoapps.student.roles import CourseInstructorRole
-from xmodule.modulestore.django import modulestore
-from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from xmodule.library_tools import LibraryToolsService
+from xmodule.modulestore.tests.factories import CourseFactory, LibraryFactory
+from xmodule.modulestore.tests.utils import MixedSplitTestCase
 from xmodule.tests import get_test_system
 from xmodule.x_module import STUDENT_VIEW  # lint-amnesty, pylint: disable=unused-import
 
+from openedx.core.djangoapps.content_libraries.tests.base import ContentLibrariesRestApiTest
+
 
-class LibrarySourcedBlockTestCase(ContentLibrariesRestApiTest):
+class LibrarySourcedBlockTestCase(MixedSplitTestCase, ContentLibrariesRestApiTest):
     """
     Tests for LibraryToolsService which interact with blockstore-based content libraries
     """
     def setUp(self):
         super().setUp()
-        self.store = modulestore()
+        self.tools = LibraryToolsService(self.store, self.user_id)
+        self.library = LibraryFactory.create(modulestore=self.store)
```

Comment:
I've noticed you added tests that cover only V1 libraries. Maybe we need to add V2 cases too?